cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoPane.tsx

```diff
@@ -14,19 +14,19 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-import { Paper } from '@mui/material';
+import { Box, Paper, SxProps } from '@mui/material';
 import { SelectedEdge } from 'bh-shared-ui';
 import React, { useState } from 'react';
 import EdgeInfoContent from 'src/views/Explore/EdgeInfo/EdgeInfoContent';
 import Header from 'src/views/Explore/EdgeInfo/EdgeInfoHeader';
-import usePaneStyles from 'src/views/Explore/InfoStyles/Pane';
+import { usePaneStyles } from 'bh-shared-ui';
 
-const EdgeInfoPane: React.FC<{ selectedEdge: SelectedEdge }> = ({ selectedEdge }) => {
+const EdgeInfoPane: React.FC<{ sx?: SxProps, selectedEdge: SelectedEdge }> = ({ sx, selectedEdge }) => {
```

Comment:
In general, I think it might have made more sense to style the container that this component lives in separately, rather than passing in this `sx` prop.

Reply:
I think this pattern is repeated a few times throughout this PR